scripts/Inelastic/CrystalField/energies.py

```diff
@@ -0,0 +1,43 @@
+from mantid.simpleapi import *
+import numpy as np
+
+
+def _unpack_complex_matrix(packed, n_rows, n_cols):
+    unpacked = np.ndarray(n_rows * n_cols, dtype=complex).reshape((n_rows, n_cols))
+    for i in range(n_rows):
+        for j in range(n_cols):
+            k = 2 * (i * n_cols + j)
+            value = complex(packed[k], packed[k + 1])
+            unpacked[i, j] = value
+    return unpacked
+
+
+def energies(nre, **kwargs):
```

Comment:
Should we add a warning that the API is not yet fixed and is subject to change? I think here for example we might want to keep consistency and upper-case the function name at least but as we haven't pinned things down yet there is probably not too much point in doing all this yet.

I also wonder if we could just have the algorithm be able to do the right thing with unpacking the matrices and not even need a warpper function/scripts but that's for later dicussion